src/components/sidebar/linkView/LinkView.tsx

```diff
@@ -65,6 +65,7 @@ class LinkView extends ViewFormBase<ILinkViewProps, ILinkViewState> {
     }
 
     async componentDidMount() {
+        super.componentDidMount();
```

Comment:
Can these calls be left out? Or is there any other better way?

Reply:
Without these, the super functions will never be called. So, this is the most elegant way I found.